docs/docsource/operation/Orbit_UI/Advanced_Workflows/Advanced_Workflows.rst

```diff
@@ -11,7 +11,7 @@ You can use Orbit to manage the following advanced workflows:
    :maxdepth: 1
    :hidden:
    
-   Setting Up Replication <Setting_Up_CRR>
+   Setting Up Replication <set_up_crr>
```

Comment:
Are we changing this to `Set Up Replication` as well?

Reply:
Done.